src/components/portfolio/DetailsMain.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { makeStyles, createStyles } from '@material-ui/core';
+import DetailsMainSummary from './DetailsMainSummary';
+import DetailsMainPositions from './DetailsMainPositions';
+import DetailsMainRisk from './DetailsMainRisk';
+import DetailsMainKeyFigures from './DetailsMainKeyFigures';
+import DetailsMainDividens from './DetailsMainDividends';
+import DetailsMainAnalyst from './DetailsMainAnalyst';
+import { RiskAnalysis, Position, KeyFigures } from './DetailsTypes';
+
+// stylesheet for the body of the details page
+const useStyles = makeStyles(() =>
+  createStyles({
+    mainWrapper: {
+      // TODO use theme margin
+      margin: '0 auto',
+      padding: '0 4rem',
+      // TODO: use theme max-width
+      maxWidth: '80rem',
+    },
+    sectionWrapper: {
+      padding: '2rem 0',
+    },
+  })
+);
+
+// type declarations
+type DetailsMainProps = {
+  positionCount: number;
+  value: number;
+  score: number;
+  perf7d: number;
+  perf1y: number;
+  risk: RiskAnalysis;
+  positions: Position[];
+  figures: KeyFigures[];
+  nextDividend: number;
+  dividendPayoutRatio: number;
+};
```

Comment:
I think we should just pass a single `PortfolioDetails` object (as received from the API) instead of having 10 properties that all contain parts of that object.

Just a note: Comments like the one on line 27 don't add any information. Docstrings on types, functions and constants whose purpose is not clear immediately are way more useful than such comments.

Reply:
@an-p Let's do that as soon as we have the actual API calls in place.